Deploy/pack_minian_run.spec

```diff
@@ -58,70 +58,11 @@ exe_minian = EXE(
     entitlements_file=None,
 )
 
```

Comment:
If I remember right  it's "Easy"

do
1. compile using pyinstaller
2. run the compiled code
3. see what error pop up (usually it say that there is a missing library)
4. add the missing library

while (compiled code not running)

And if you are not lucky it's a guess..... (like the time when minian was running on my PC and yours but not on other because a librabry was not fully copied and only linked in the bundling....)

See "easy"......

Reply:
Oki, I was hopping there was a better way.. I was doing the same, but I guess was less lucky/productive 😂

Reply:
On what I remember there is not really an other way.... and it was just a pain. The only thing is at the end I just did a `collect_all( )` on the library and that solved a lot of problem too